docs/Available-Symbols-Generators.md

```diff
@@ -305,10 +281,10 @@ This sample creates different symbols showing the different formatting available
 Creates a symbol from the current date/time. 
 
 #### Parameters 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|format|string|DateTime.ToString format|
-|utc|bool|UTC time if true, local time otherwise|    
+|Name|Data Type|Description|Mandatory|
+|----------|------|---------------|---|
+|`format`|`string`|`DateTime.ToString()` format|no|
```

Comment:
Do we want to link msdne here? E.g. https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings